lib/rest-adapter.js

```diff
@@ -369,22 +369,40 @@ RestAdapter.errorHandler = function(options) {
         res.statusCode = err.statusCode || err.status || 500;
       }
 
-      debug('Error in %s %s: %s', req.method, req.url, err.stack);
-      var data = {
-        name: err.name,
-        status: res.statusCode,
-        message: err.message || 'An unknown error occurred',
-      };
-
-      for (var prop in err) {
-        data[prop] = err[prop];
+      if (Array.isArray(err)) {
+        var details = err.map(function(it) {
+          var data = generateResponseError(it);
+          delete data.statusCode;
+          return data;
+        });
+
+        err =
+          new Error('Failed with multiple errors, see `details` for more information.');
```

Comment:
@bajtos, I will not ignore this comment! Haha, I completely agree. While I'm not a *huge* fan of your first suggestion, I do prefer your second. I usually do that, not sure why I didn't this time.

Regardless, I fixed it